pages/issue.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
+import { Helmet } from "react-helmet";
 import IssueSelector from '../components/issue-selector/issue-selector.jsx';
 import ProjectLoader from '../components/project-loader/project-loader.jsx';
 
 export default function (props) {
-  const encodeIssueAsUri = function(issue) {
+  const getFullIssueName = function(issue) {
```

Comment:
there is no scope busting going on in this function, so we can declare it outside of the export instead of inlining it here.

We can also improve this code because we don't even need a function here:
```
const mapping = {
  "online-privacy-and-security": "Online Privacy & Security",
  "open-innovation": "Open Innovation",
  ...
}

export default function (props) {
  const issueName = mapping[props.params.issue] || ``;

  return <div>
            <Helmet><title>{issueName}</title></Helmet>
            <IssueSelector />
            <ProjectLoader issue={encodeURIComponent(issueName)} />
          </div>;
};
```